algebird-core/src/main/scala/com/twitter/algebird/Ring.scala

```diff
@@ -243,19 +243,19 @@ object Ring extends GeneratedRingImplicits with ProductRings with RingImplicits0
     else Some(rng.product(it))
 
   implicit val unitRing: Ring[Unit] = new ConstantRing(())
-  implicit def boolRing: Ring[Boolean] = BooleanRing
-  implicit def jboolRing: Ring[JBool] = JBoolRing
-  implicit def intRing: Ring[Int] = IntRing
-  implicit def jintRing: Ring[JInt] = JIntRing
-  implicit def shortRing: Ring[Short] = ShortRing
-  implicit def jshortRing: Ring[JShort] = JShortRing
-  implicit def longRing: Ring[Long] = LongRing
-  implicit def bigIntRing: Ring[BigInt] = BigIntRing
-  implicit def jlongRing: Ring[JLong] = JLongRing
-  implicit def floatRing: Ring[Float] = FloatRing
-  implicit def jfloatRing: Ring[JFloat] = JFloatRing
-  implicit def doubleRing: Ring[Double] = DoubleRing
-  implicit def jdoubleRing: Ring[JDouble] = JDoubleRing
+  implicit val boolRing: Ring[Boolean] = BooleanRing
```

Comment:
why change these to `val`? `def` is just as good since these are all objects that have already been allocated, no?

Reply:
Ahh oops! yup in this case it's the same thing; I was doing this initially to propose a consistent way to define implicit's;

I'll revert this!